src/handlers/initialize.rs

```diff
@@ -17,15 +17,18 @@ pub async fn initialize(_backend: &Backend, params: InitializeParams) -> Result<
 
     Ok(InitializeResult {
         capabilities: SERVER_CAPABILITIES.clone(),
-        ..Default::default()
+        server_info: Some(ServerInfo {
+            name: String::from("ts_query_ls"),
+            version: Some(String::from("1.4.0")),
```

Comment:
What about this for the `version` field?

```rust
env!("CARGO_PKG_VERSION").to_string()
```

Reply:
Nice thx!